features/call/impl/src/main/kotlin/io/element/android/features/call/impl/utils/ActiveCallManager.kt

```diff
@@ -83,44 +90,70 @@ class DefaultActiveCallManager @Inject constructor(
 ) : ActiveCallManager {
     private var timedOutCallJob: Job? = null
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal val activeWakeLock: PowerManager.WakeLock? = run {
+        val powerManager = context.getSystemService<PowerManager>()
+
+        if (powerManager?.isWakeLockLevelSupported(PowerManager.PARTIAL_WAKE_LOCK) == true) {
+            powerManager.newWakeLock(
+                PowerManager.PARTIAL_WAKE_LOCK,
+                "${context.packageName}:IncomingCallWakeLock",
+            )
+        } else {
+            null
+        }
+    }
```

Comment:
Could be simplified as:
```kotlin
    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
    internal val activeWakeLock: PowerManager.WakeLock? = context.getSystemService<PowerManager>()
        ?.takeIf { it.isWakeLockLevelSupported(PowerManager.PARTIAL_WAKE_LOCK) }
        ?.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, "${context.packageName}:IncomingCallWakeLock")
```